README.md

```diff
@@ -602,6 +601,15 @@ VPC CNI uses `iptables-legacy` by default. Setting `ENABLE_NFTABLES` to `true` w
 **Note:** VPC CNI image contains `iptables-legacy` and `iptables-nft`. Switching between them is done via `update-alternatives`. It is *strongly* recommended that the iptables mode matches that which is used by the base OS and `kube-proxy`.
 Switching modes while pods are running or rules are installed will not trigger reconciliation. It is recommended that rules are manually updated or nodes are drained and cordoned before updating. If reloading node, ensure that previous rules are not set to be persisted.
 
+#### `AWS_EXTERNAL_SERVICE_CIDRS` (v1.13.0+)
+
+Type: String
+
+Default: empty
+
+Specify a comma-separated list of IPv4 and/or IPv6 CIDRs that *must* be routed via main routing table. This is required for secondary ENIs to reach endpoints outside of VPC that are backed by a service.
+For every item in the list, an `ip rule` will be created with a priority greater than the from container rule. If an item is not a valid IPv4 or IPv6 CIDR, it will be skipped.
```

Comment:
nit: `from container rule` is probably too technical for someone who is not familiar with the code base. Maybe we can change it to `priority greater than ip rule capturing the egress traffic from the container` or something similar..